app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,30 @@
+module SessionsHelper
+
+  def log_in(admin_user)
+    session[:adminuser_id] = admin_user.id
```

Comment:
セッションのキー名も `adminuser_id` ではなく `admin_user_id` がいいですね 👍 
ここ以外も全体的に修正してみよう〜！
```suggestion
    session[:admin_user_id] = admin_user.id
```